faqs/galaxy/account_update_email_username_or_password.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Update email, username, or password
+area: account
+box_type: tip
+layout: faq
+contributors: [jennaj, Melkeb]
+---
+
+
+1. Go to the Galaxy server you are using
+2. Log in under User > Login (menu located in the top masthead)
+3. Once logged in, go to User > Preferences (the menu updates after login)
+4. Click on the links to reach forms to make the changes
+5. Some changes will require email confirmation. Be sure to use a valid email address
```

Comment:
This FAQ only covers changing name, email address and password. Of those, only changing the email address requires confirmation. Maybe we can reword this FAQ a bit more clearly?